ttl/cache/table.go

```diff
@@ -132,6 +132,10 @@ func NewPhysicalTable(schema model.CIStr, tbl *model.TableInfo, partition model.
 	}, nil
 }
 
+func isTTLTable(tbl *model.TableInfo) bool {
+	return tbl.TTLInfo != nil && tbl.State == model.StatePublic
```

Comment:
we can make it as a member function of ```model.TableInfo```.

Reply:
If I see a function named `IsTTLTable` in `model`, I would expect it to be simply `tbl.TTLInfo != nil` without comparing the `tbl.State`.

As this function is only used by one function, I choose to inline it to give the condition more context and make it clearer. (So this function doesn't exist now :beers:)